fe/fe-core/src/main/java/org/apache/doris/catalog/Table.java

```diff
@@ -649,4 +627,9 @@ public Map<String, Set<String>> findReAnalyzeNeededPartitions() {
     public List<Long> getChunkSizes() {
         throw new NotImplementedException("getChunkSized not implemented");
     }
+
+    @Override
+    public long fetchRowCount() {
+        throw new NotImplementedException("Internal table doesn't implement fetchRowCount");
```

Comment:
How about implement this method as `getRowCount()`, and in `getRowCount()`, call `fetchRowCount()` directly?
So that the meaning of `fetchRowCount()` is same among all kinds of tables: fetch the row count.

Reply:
Will fix